doc/source/commands/keg.rst

```diff
@@ -28,14 +28,14 @@ are situations where many image descriptions must be managed and the
 image descriptions have considerable overlap with respect to content
 and setup.
 
-The key component for Keg is a data structure called `image definition tree`.
+The key component for Keg is a data structure called `recipes`.
```

Comment:
I replaced `image definition tree` because we didn't really use it at any point and `recipes` seems the established term for keg input data. The sentence does sound a bit clumsy now though. I'll rephrase.